python/lsst/ts/criopy/gui/sal/application.py

```diff
@@ -110,10 +132,17 @@ def add_comm(
         """
         self._comms_args.append(SplashScreen.CommArgs(name, manual, kwargs))
 
+    def process_command_line(self) -> None:
+        """Called to process command line arguments. Shall be overridden in
+        children, if custom command line processing is desired. Use self.parser
+        to access command line parser."""
+        pass
```

Comment:
How about to `raise` the `NotImplementedError` instead?

Reply:
It's perfectly legal to not process custom command line arguments if there aren't any custom command line arguments. This is a placeholder for children to override.